packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -190,12 +226,19 @@ if (!ExecutionEnvironment.canUseDOM) {
     callback: (deadline: Deadline) => void,
     options?: {timeout: number},
   ): number {
-    // This assumes that we only schedule one callback at a time because that's
-    // how Fiber uses it.
-    scheduledRICCallback = callback;
+    let timeoutTime = -1;
     if (options != null && typeof options.timeout === 'number') {
       timeoutTime = now() + options.timeout;
     }
+    // This assumes that we only schedule one callback at a time because that's
+    // how Fiber uses it.
```

Comment:
I think this comment should be above `` rather than the call to `getCallbackId`.

But also, the comment confuses me a little. A lot of this module seems written to support multiple, concurrent callbacks (like the fact that rIC returns an id and adds to a `registeredCallbackIds` map).

Reply:
It is laying the groundwork for support to multiple callbacks, yes. I have a follow-up PR that implements that support, just breaking things into multiple steps.

Reply:
Regarding the comment - it's outdated, will remove either in this PR or the next.